src/backend/expungeservice/charge_classifier.py

```diff
@@ -238,19 +241,27 @@ def _handle_pcs_and_manufacture_delivery(name, level, statute, schedule_2_handle
             question_string = "Was the underlying substance marijuana?"
             charge_types_with_question = ChargeClassifier._classification_by_level(level, statute)
             if level == "felony unclassified":
-                felony_unclassified_question_id = (
+                yes_question_string= "Was the charge for an A Felony, B Felony, or C Felony?"
+                yes_question_id = f"{question_string}-Yes-{yes_question_string}"
+                options_dict = {
+                    "A Felony": Answer(edit={"charge_type": "MarijuanaEligibleFelonyClassA"}),
+                    "B Felony": Answer(edit={"charge_type": "MarijuanaEligibleFelonyClassB"}),
+                    "C Felony": Answer(edit={"charge_type": "MarijuanaEligibleFelonyClassC"}),
+                }
+                yes_question = Question(yes_question_id, yes_question_string, options_dict)
+                no_question_id = (
                     f"{question_string}-No-{charge_types_with_question.question.question_id}"
                 )
-                felony_unclassified_question = replace(
-                    charge_types_with_question.question, question_id=felony_unclassified_question_id
+                no_question = replace(
+                    charge_types_with_question.question, question_id=no_question_id
                 )
-                charge_types = [MarijuanaEligible()] + charge_types_with_question.ambiguous_charge_type
+                charge_types = [MarijuanaEligible(severity_level="Felony Class A"),  MarijuanaEligible(severity_level="Felony Class B"),MarijuanaEligible(severity_level="Felony Class C")] + charge_types_with_question.ambiguous_charge_type
```

Comment:
These should probably use the new classes too then. Also see if you can refactor to use the method at the bottom of the file.